video2numpy/read_vids_cv2.py

```diff
@@ -6,21 +6,38 @@
 from .resizer import Resizer
 from .shared_queue import SharedQueue
 from .utils import handle_youtube
+from .utils import extract_audio_from_url
 
 
-def read_vids(vids, worker_id, take_every_nth, resize_size, batch_size, queue_export):
+def read_vids(vids, 
+             worker_id, 
+             take_every_nth, 
+             resize_size, 
+             batch_size, 
+             queue_export_video,
+             queue_export_audio,
+             output_dir,
+             no_audio,
```

Comment:
Perhaps change this to one parameter "modalities" and have it be a list of strings with a default of ["frames", "audio"]
this way later we don't need to add another parameter for metadata we can just add another string.

In the read_vids function you can just check
```
if "audio" in modalities:
    do something
```

Reply:
Ok, already done.